### ISSUE TITLE
Doesn't work on 'fresh' fedora 23

### ISSUE THREAD

**Issue by ygwain** [iacopodeenosee/EasyScreenCast#63]
When I click to 'Start record' nothing happens.

**Comment by fastrizwaan** [comment]
same with me, If I choose "Audio source" and click on "Start Recording" it gives error : "ERROR RECORDER - see logs for more..

Where are the logs btw?

**Comment by iacopodeenosee** [comment]
> Where are the logs btw?

you must **enable** the verbose logs in options window and after that you can find the logs by typing this on terminal -> `journalctl --since=today --no-pager | grep js`
paste here the result, thx

**Comment by fastrizwaan** [comment]
[output.txt](https://github.com/iacopodeenosee/EasyScreenCast/files/185978/output.txt)

**Comment by fastrizwaan** [comment]
OK, working with "no audio source" and "Default Audio Source" but it is not recording the system sound, say I'm recording audio playback and video playback on my system, which is not being recorded.

Choosing different audio source causes the error. Also choosing mkv and high bitrate causes error.


![untitled](https://cloud.githubusercontent.com/assets/1161682/13985836/fc401952-f124-11e5-81a6-9b6a9844042f.png)

**Comment by ygwain** [comment]
How often does it update on extensions.gnome.org site?

**Comment by iacopodeenosee** [comment]
with each release is sent for review and publication on extensions.gnome.org site, if you do not want to wait you can install it by downloading it directly from here:
https://github.com/iacopodeenosee/EasyScreenCast/archive/0.9.8.1.zip

**Comment by ygwain** [comment]
Still does not work for me and seems like log is not helpful:
```
Mar 26 21:16:36 alex-pc.lan gnome-shell.desktop[1945]: EasyScreenCast_Indicator<._doDelayAction@/home/alex/.local/share/gnome-shell/extensions/EasyScreenCast@iacopodeenosee.gmail.com/extension.js:440
Mar 26 21:16:36 alex-pc.lan gnome-shell.desktop[1945]: EasyScreenCast_Indicator<._doRecording@/home/alex/.local/share/gnome-shell/extensions/EasyScreenCast@iacopodeenosee.gmail.com/extension.js:468
Mar 26 21:16:36 alex-pc.lan gnome-shell.desktop[1945]: EasyScreenCast_Indicator<._addMIRecording/<@/home/alex/.local/share/gnome-shell/extensions/EasyScreenCast@iacopodeenosee.gmail.com/extension.js:175

```
The simplest way to reproduce I believe it's to boot from Fedora 23 live media and install the extension (I'm trying under freshly installed distro to HDD).

**Comment by iacopodeenosee** [comment]
@ygwain can you post the entire log?
- restart gnome shell
- try to start screencasting in different ways (audio, delay, webcam) so it is easier to figure out where is the problem

> you must enable the verbose logs in options window and after that you can find the logs by typing this on terminal -> journalctl --since=today --no-pager | grep js
paste here the result, thx

**Comment by ygwain** [comment]
http://pastebin.com/gcLuV0eS
When I set audio to "no audio" explicitly (despite it set by default) it does work, but if I restore default options it doesn't work again.

**Comment by ygwain** [comment]
For version installed from github master:
http://pastebin.com/EDVUtmE0
With defaults settings nothing happens, after I set audio source to any option it works. Seems like something wrong with default options.

**Comment by iacopodeenosee** [comment]
@ygwain hi, in the first log i see the bug i previously I solved, in the second one i can see the bug you described:
```
    getAudioSource: function() {
        Lib.TalkativeLog('-#-get source audio choosen');

        var arrtmp = this.getListInputAudio();
        var index = Pref.getOption(
            'i', Pref.INPUT_AUDIO_SOURCE_SETTING_KEY) - 2;
        return arrtmp[index].name;
    },
```
if you can send me the complete log(_enable the verbose logs in options window_) I have more chance of solving. thx

**Comment by ygwain** [comment]
http://pastebin.com/7Hg7H30Q

**Comment by iacopodeenosee** [comment]
thx for logs, i'll try ESC in a live fedora 23 to reproduce the bug

**Comment by iacopodeenosee** [comment]
@ygwain I think the problem has been solved with this issue #66

**Comment by ygwain** [comment]
Seems like fixed, but it was removed while updating via extensions.gnome.org so I have installed it again. Thank you very much for a great extension!
